From: Carlos Maddela <e7appew@gmail.com>
Date: Tue, 14 Feb 2017 08:16:30 +1100
Subject: Cherry-pick missing upstream changelog entry for version 2.4.6.

Description: Cherry-pick missing upstream changelog entry for version 2.4.6.
Origin: upstream, https://github.com/sahib/rmlint/commit/e9e0363446dab27833e3b23ba3d87991b6f87355
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 CHANGELOG.md | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/CHANGELOG.md b/CHANGELOG.md
index 7e560f3..c0ed4e5 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,17 @@ The format follows [keepachangelog.com]. Please stick to it.
 
 ## [2.5.0 Penetrating Pineapple] -- [unreleased]
 
+## [2.4.6 Myopic Micrathene Bugfixes] -- 2017-01-16
+
+### Fixed
+
+* Update translation template.
+* Various fix to make rmlint ready to be packaged in debian.
+
+### Added
+
+* Output of rmlint.sh uses a colored prefix now.
+
 ## [2.4.5 Myopic Micrathene Bugfixes] -- 2016-12-12
 
 ### Fixed
